pkgs/development/tools/coder/default.nix

```diff
@@ -14,18 +24,58 @@ buildGoModule rec {
     hash = "sha256-26RvDJ890MclDB4rtYQ7CcB3NQRXC7sI2cXd689Eq6E=";
   };
 
-  # integration tests require network access
-  doCheck = false;
+  offlineCache = fetchYarnDeps {
+    yarnLock = src + "/site/yarn.lock";
+    hash = "sha256-xWHLllnJOivjxS56Hp9QhV7oNQhW6TROrzIN1B77eKE=";
+  };
+in
+buildGoModule rec {
+  inherit pname version src;
```

Comment:
I am not opposed to inheriting here, but I think you will get push back from others, as they generally prefer rec to inherit

Reply:
Do you have an example what you mean 🙈

Reply:
this will work, and if the final reviewer signs off, fine by me; but I am expecting they will want something like:
```nix
buildGoModule rec {
  pname = "coder";
  version = "0.13.3";

  src = fetchFromGitHub {
    owner = pname;
    repo = pname;
    rev = "v${version}";
    hash = "sha256-26RvDJ890MclDB4rtYQ7CcB3NQRXC7sI2cXd689Eq6E=";
  };

  offlineCache = fetchYarnDeps {
    yarnLock = src + "/site/yarn.lock";
    hash = "sha256-xWHLllnJOivjxS56Hp9QhV7oNQhW6TROrzIN1B77eKE=";
  };
```

Reply:
Adopted. I didn't know that you can directly reference src a line below it :).

Many thanks for your review, learned more Nix tricks and felt a little bit stupid asking :)

Reply:
> https://nixos.org/manual/nix/stable/language/constructs.html#recursive-sets

yeah, they are really cool; and it has been my pleasure!